pybliometrics/scopus/serial_title.py

```diff
@@ -193,7 +212,8 @@ def __str__(self):
 def _parse_list(d, list):
     """Auxiliary function to parse SNIP and SJR lists."""
     keyword = list + "List"
-    try:
-        return (d[keyword][list][0]['@year'], d[keyword][list][0]['$'])
-    except KeyError:
-        return None
+    if d[keyword] is None:
+        return 'Not available'
+    else:
+        return [ (r['@year'], r['$']) for r in d[keyword][list] ]
+
```

Comment:
Why this change? In general, it's pybliometrics' philosophy to return `None` if that's what the API returns. It's also a cheaper on resources and faster. Same goes for try-except clauses, if used in a smart way. So I think we can just put the list comprehension in the try-part, no?
Also, since you're using sorted whenever you use this function, it's wise to use the sorted already here and spare it above. This avoids code duplication. And finally please drop the surrounding blanks (inside the square brackets), that's not what PEP8 suggests

Reply:
you are right and sorry I am not so used to PEP8. Will change that accordingly